components/eam/src/control/scamMod.F90

```diff
@@ -198,9 +207,12 @@ subroutine scam_default_opts( scmlat_out,scmlon_out,iopfile_out, &
    if ( present(iop_nudge_tq_low_out) ) iop_nudge_tq_low_out = 1050.0_r8
    if ( present(iop_nudge_tq_high_out) ) iop_nudge_tq_high_out = 0.e3_r8
    if ( present(iop_nudge_tscale_out) ) iop_nudge_tscale_out = 10800.0_r8
+   if ( present(iop_perturb_high_out) ) iop_perturb_high_out = 1050.0_r8
    if ( present(scm_crm_mode_out) )     scm_crm_mode_out  = .false.
```

Comment:
I know this is off topic, but is it correct that some of these "crm" options are no longer working/supported?
I've seen other old "crm" references in the physics code, which I've never really understood how they were used. Would it be worth cleaning those up in a future PR?

Reply:
@whannah1 I was on the fence about removing those in this PR.  I have also never been clear about what these options do and have not heard of one soul that uses them in my five years of being the SCM contact-point.  I will likely reconsider removing them in this PR.

Reply:
ok sounds good!